idowuadeleke/BANKA
# #165551853 add validation middleware

### What does this PR do?
Add validation middleware

#### Description of Task to be completed?
- replace controller validation with validation middleware

#### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-165551853-create-validation-middleware`

- all input validation should be only done inside validation middleware

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#165551853](https://www.pivotaltracker.com/story/show/165551853)

#### Screenshots (if appropriate)
N/A
